modules/470-chrony/images/chrony-exporter/werf.inc.yaml

```diff
@@ -0,0 +1,72 @@
+{{- $chronyExproterVersion := "0.11.0"}}
+{{- $promuVersion := "0.14.0"}}
+---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact-promu
+from: {{ $.Images.BASE_GOLANG_23_ALPINE_DEV }}
```

Comment:
please do not use _DEV images

use $.Images.BASE_GOLANG_23_ALPINE

Reply:
sorry

src-artifact

https://github.com/deckhouse/deckhouse/tree/main/modules/000-common/images/src-artifact

Reply:
for src-artifact images